mlflow/tracking/service.py

```diff
@@ -135,24 +135,26 @@ def set_tag(self, run_id, key, value):
         tag = RunTag(key, str(value))
         self.store.set_tag(run_id, tag)
 
-    def log_artifact(self, artifact_uri, local_path, artifact_path=None):
+    def log_artifact(self, run_id, local_path, artifact_path=None):
```

Comment:
The changes here were made in master and I think you accidentally reverted them at some point.